devicetypes/smartthings/ct100-thermostat.src/ct100-thermostat.groovy

```diff
@@ -21,8 +21,8 @@ metadata {
 		command "poll"
 
 		fingerprint deviceId: "0x08", inClusters: "0x43,0x40,0x44,0x31,0x80,0x85,0x60"
-		fingerprint mfr:"0098", prod:"6401", model:"0107", deviceJoinName: "2Gig CT100 Programmable Thermostat"
-		fingerprint mfr:"0098", prod:"6501", model:"000C", deviceJoinName: "Radio Thermostat CT101"
+		fingerprint mfr:"0098", prod:"6401", model:"0107", deviceJoinName: "2Gig Thermostat" //2Gig CT100 Programmable Thermostat
+		fingerprint mfr:"0098", prod:"6501", model:"000C", deviceJoinName: "2Gig Thermostat" //Radio Thermostat CT101
```

Comment:
I believe the manufacturer name was left off of here on purpose because this product is sold under a different manufacturer. I could be wrong but that should be verified here and in any other fingerprint where the manufacturer is not currently present.

Reply:
Tom is correct. This (and most of the thermostat) are white labeled and sold by other brands.